.jenkins

```diff
@@ -183,6 +183,56 @@ pipeline {
                         }
                     }
                 }
+                stage('CUDA-11.0-NVCC-C++17-RDC-KCL') {
+                    agent {
+                        dockerfile {
+                            filename 'Dockerfile.nvcc'
+                            dir 'scripts/docker'
+                            additionalBuildArgs '--pull --build-arg BASE=nvidia/cuda:11.0-devel-rc --build-arg ADDITIONAL_PACKAGES="g++-8" --build-arg CMAKE_VERSION=3.17.3'
+                            label 'nvidia-docker && volta'
+                            args '-v /tmp/ccache.kokkos:/tmp/ccache'
+                        }
+                    }
+                    environment {
+                        OMP_NUM_THREADS = 8
+                        OMP_PLACES = 'threads'
+                        OMP_PROC_BIND = 'spread'
+                        NVCC_WRAPPER_DEFAULT_COMPILER = 'g++-8'
```

Comment:
```
cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS=-Werror -DCMAKE_CXX_STANDARD=17 -DKokkos_ENABLE_COMPILER_WARNINGS=ON -DKokkos_ENABLE_OPENMP=ON -DKokkos_ENABLE_CUDA=ON -DKokkos_ENABLE_CUDA_LAMBDA=OFF -DKokkos_ENABLE_CUDA_UVM=ON -DKokkos_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE=ON -DKokkos_ARCH_VOLTA70=ON -DCMAKE_INSTALL_PREFIX=/var/jenkins/workspace/Kokkos/build/../install ..
-- Setting default Kokkos CXX standard to 17
-- The CXX compiler identification is GNU 7.5.0
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ - works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Setting policy CMP0074 to use <Package>_ROOT variables
-- The project name is: Kokkos
-- Compiler Version: 11.0.167
-- kokkos_launch_compiler (/var/jenkins/workspace/Kokkos/bin/kokkos_launch_compiler) is enabled... Set Kokkos_LAUNCH_COMPILER_INFO=ON for more info.
-- Using -std=c++1z for C++17 standard as feature
CMake Error at cmake/kokkos_test_cxx_std.cmake:33 (MESSAGE):
  CMake wants to use -std=c++1z which is not supported by NVCC.  Using a more
  recent host compiler or a more recent CMake version might help.
Call Stack (most recent call first):
  cmake/kokkos_test_cxx_std.cmake:75 (kokkos_set_cxx_standard_feature)
  cmake/kokkos_tribits.cmake:201 (INCLUDE)
  CMakeLists.txt:161 (KOKKOS_SETUP_BUILD_ENVIRONMENT)
```
Apparently, `gcc-7.5.0` is still used for feature detection.

Reply:
Yea, just fixed that, it was setting `NVCC_WRAPPER_DEFAULT_COMPILER` instead of `CXX` in the environment